onecodex/models/collection.py

```diff
@@ -350,7 +353,10 @@ def _collate_results(self, metric=None, include_host=None):
                 if not include_host and d_tax_id in host_tax_ids:
                     continue
 
-                data[c_idx, tax_id_to_idx[d_tax_id]] = d[metric] or 0
+                if metric in [Metric.AbundanceWChildren, Metric.Abundance]:
```

Comment:
I don't think you need this check anymore after lines 342-343. The default value should already be `0` for metrics other than `AbundanceWChildren` and `Abundance`

Reply:
The value in d[metric] would still be possibly `None` though, so as far as I can see we either need this check or to replace what we have with something like
```
if d[metric]:
    data[c_idx, tax_id_to_idx[d_tax_id]] = d[metric]
```
Maybe that's clearer ^ - wdyt?

Reply:
oops. You're right. I think `d[metric] or 0` is clearer.